packages/edit-site/src/components/style-book/constants.ts

```diff
@@ -247,7 +252,7 @@ export const STYLE_BOOK_IFRAME_STYLES = `
 	}
 
 	.edit-site-style-book__example-preview {
-		width: 100%;
+		width: calc(100% - 120px);
```

Comment:
It looks like this results in the reduced width in narrower sizes:

| Before this rule | With this rule |
| --- | --- |
| <img width="545" alt="image" src="https://github.com/user-attachments/assets/c02e31b7-2465-4b7a-8c97-7d445fd7102e"> | <img width="523" alt="image" src="https://github.com/user-attachments/assets/6ef9b8ab-d4b6-4929-849c-643705ab712b"> |

Would it be worth using a media query or other way to guard this?

Reply:
Done!